<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>12</y>
      <width>201</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="useIndexButton">
       <property name="text">
        <string>Do not use index</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Display height maps:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="originalImageButton">
       <property name="text">
        <string>Original image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="smoothedImageButton">
       <property name="text">
        <string>Smoothed image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="gradientNormButton">
       <property name="text">
        <string>Gradient norm image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cannyImageButton">
       <property name="text">
        <string>Canny image</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>201</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTextBrowser" name="errorText"/>
     </item>
     <item>
      <widget class="QTextBrowser" name="imageFileText"/>
     </item>
     <item>
      <widget class="QPushButton" name="choseImageButton">
       <property name="text">
        <string>Chose another image</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>421</width>
      <height>181</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Instructions&lt;/span&gt;  : &lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;First, chose another image or keep the demo image. it must be a greyscale image. Then, launch visualisations of heightmaps corresponding to any  step of Canny algorithm.&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;To control the display, use &lt;span style=&quot; font-weight:600;&quot;&gt;ZQSD&lt;/span&gt; to rotate the model, arrows to rotate the light source, &lt;span style=&quot; font-weight:600;&quot;&gt;space bar&lt;/span&gt; to make the plan appear or disappear, &lt;span style=&quot; font-weight:600;&quot;&gt;RF&lt;/span&gt; to raise or lower it and &lt;span style=&quot; font-weight:600;&quot;&gt;W&lt;/span&gt; to save the current rendering as an image.&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;It is also possible to disable and enable the use of the index with &amp;quot;Do not use index&amp;quot; and &amp;quot;Use index&amp;quot;. disabling index could be usefull with really sharp images, such as images resulting from Canny algorithm.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
